### ISSUE TITLE
Implement Vault credential management

### ISSUE THREAD

**Issue by timothydnguyen** [approzium/approzium#4]
how will creds be stored in and retrieved from vault? are they stored against ARN as key?

**Comment by timothydnguyen** [comment]
requirements
- include vault setup in environment
- authenticator retrieves password stored against (dbhost (target), arn (identity)) key

**Comment by timothydnguyen** [comment]
/\_\_approzium\_\_<DB_ADDRESS>
```
key: "dbuser1"
value: {
	"password": "asdfghjkl",
	"IAM_roles": [
		"arn:aws:iam::accountid:role/rolename1", 
		"arn:aws:iam::accountid:role/rolename2"
	]
}
```

**Comment by tyrannosaurus-becks** [comment]
Thanks for including this! Also, from my notes, I will be sure to include the db port, and there could be multiple users.

**Comment by tyrannosaurus-becks** [comment]
Closed by #35
